test/models/like_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+
+class LikeTest < ActiveSupport::TestCase
+  def setup
+    @post = posts(:valid)
+  end
+
+  test 'create a like' do
+    assert @post.valid?
+    assert @post.save
```

Comment:
Asserting that a post in your fixtures is valid is a bit unnecessary. Everything in your fixtures should be valid by default, if it's not then you've got a problem.